19.md

```diff
@@ -35,18 +35,25 @@ These are the possible bech32 prefixes with `TLV`:
   - `nprofile`: a nostr profile
   - `nevent`: a nostr event
   - `nrelay`: a nostr relay
+  - `nitem`: a nostr parameterized replaceable event coordinate (NIP-33)
```

Comment:
`item` is such a generic word. I've been trying to come up with a better one to suggest, but I'm not sure there is one haha. I'm guessing you had the same thought.

If an event is a representation of some data at particular point in time, then the "d" tag in the replaceable event is the address of a place where we can find the latest data. `naddress` seems almost as generic as `nitem`. `nregister` (like a CPU register) means a similar thing but is a little less generic. 

Is the "d" from NIP-33 short for something? Maybe there is some inspiration there.

Reply:
I don't know. @Cameri please help. Also please offer your opinion here.

Reply:
the `d` stands for deduplication identifier.
I think it should be `nreg` with the kind (30000 to 39999), tag index, and pubkey.

The big issue is that the link can be broken easily by someone changing the replaceable event so I'm almost inclined to include a checksum or partial hash of the event instead of index.

Reply:
`nid`?

Reply:
> I agree "register" makes sense, but `nreg` is such an ugly word, probably `naddr` is better.

register sounds good to me!

Reply:
`nref` ?

Reply:
> nref

full circle :laughing: